src/validations/rules/ssp.sch

```diff
@@ -696,15 +697,19 @@
                 value="substring-before(@control-id, '-')" />
             <sch:let
                 name="leveragedUUID"
-                value="prop[@name = 'leveraged-authorization-uuid']/@value" />
+                value="oscal:prop[@name = 'leveraged-authorization-uuid']/@value" />
             <sch:assert
                 diagnostics="leveraged-PE-controls-implemented-requirement-diagnostic"
                 id="leveraged-PE-controls-implemented-requirement"
                 role="warning"
                 test="
-                    if ($leveraged/@uuid eq $leveragedUUID and $familyName eq 'pe')
+                    if ($familyName eq 'pe')
                     then
-                        false()
+                        if ($leveraged/@uuid eq $leveragedUUID)
+                        then
+                            (true())
+                        else
+                            (false())
                     else
                         true()">This PE Control has a leveraged authorization - <xsl:value-of
                     select="@control-id" />.</sch:assert>
```

Comment:
This assertion's diagnostic message text is "There are PE controls inherited from leveraged authorizations." That does not explain the **negative** test outcome.